### PR TITLE
Initial copy of performance and performance-types to packages-exp

### PR DESCRIPTION
**Pull request by jposuna**
There are barely any changes compared to the standard performance package. Only changed the relevant information in the package.json for both folders. I'll add changes incrementally in future PRs.

### CODE REVIEW

**Review by alikn — APPROVED** [review]

File: packages-exp/performance-exp/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# @firebase/performance
```
Comment (hsubox76) [inline packages-exp/performance-exp/CHANGELOG.md:1]: Probably don't want to copy this file.

File: packages-exp/performance-exp/README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# @firebase/performance
```
Comment (hsubox76) [inline packages-exp/performance-exp/README.md:1]: add "-exp"

File: packages-exp/performance-exp/index.ts [hunk 3]
```diff
@@ -0,0 +1,83 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC
```
Comment (hsubox76) [inline packages-exp/performance-exp/index.ts:3]: Update years in copyright comments, here and elsewhere.

File: packages-exp/performance-exp/package.json [hunk 4]
```diff
@@ -0,0 +1,59 @@
+{
+  "name": "@firebase/performance-exp",
+  "version": "0.4.1",
```
Comment (hsubox76) [inline packages-exp/performance-exp/package.json:3]: We're setting this to "0.0.800" for all -exp packages.

File: packages-exp/performance-types-exp/package.json [hunk 5]
```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@firebase/performance-types-exp",
+  "version": "0.0.13",
```
Comment (hsubox76) [inline packages-exp/performance-types-exp/package.json:3]: "0.0.800"

**Review by hsubox76 — CHANGES_REQUESTED** [review]

File: packages-exp/performance-exp/rollup.config.js [hunk 6]
```diff
@@ -0,0 +1,69 @@
+/**
+ * @license
+ * Copyright 2018 Google LLC
```
Comment (hsubox76) [inline packages-exp/performance-exp/rollup.config.js:3]: I think this one got missed.

**Review by hsubox76 — APPROVED** [review]

**Pull request merged**